### PR TITLE
Project 4: Apple clone web page

### PR DESCRIPTION
**Pull request by enaburekhan**
**Apple original web page cloned by Eric Enaburekhan and Tendai Nyandoro.**
In this project, we:

- use images as a background using flex
- Add gradients to elements using CSS
- use flex, grid and float for positioning.
- The goal was to make a clone of an old version of Apple website webpage

### CODE REVIEW

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Invalid 🛑 

## This project cannot be reviewed without the linter checks, please add the [linters config](https://github.com/microverseinc/linters-config) and ask for another review


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Hi, you need more work :hammer: :wrench: :toolbox: to get it much better :100: :+1: , please fix the issues bellow.

- [ ] Fix the linter errors:
image

- [ ] You should not use images over text:
image

- [ ] fix the search input it's not well designed:
image

- [ ] Fix the text inside the navbar items they should be perfectly centered.
image

- [ ] the main image should cover the whole page. and it should be set on the background of the page. you can use background size and position properties to achieve that.

- [ ] the four images below the main image should be all in one line, each one of them should take 25% of the width.
image


- [ ] Fix the footer, make the text smaller and have the same color as in the original:

|*Clone*|*Original*|
|:---:|:---:|
| image | image |

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat:

**Comment by enaburekhan** [comment]
**Code review:**

- Fix footer link color
- Fix input search bar
- They are images and not text 
- The images are well aligned as seen in the read me and attach here
- Fix linter issue
- Fix text alignment inside nav bar

main-img

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    <main>
+        <figure class="margin width-100 center dflex main1">
+            <img src="images/header_title.png" alt="text message"> 
+        </figure>
+        <figure class="margin width-100 center dflex sub-main1">
+            <img src="images/header_intro.png" alt="text message"> 
+        </figure>  
+        <figure class="margin width-100 center dflex bg-img">
+        <img src="images/hero.jpg" alt="hero expedition image"> 
+        </figure>
```
Comment (SpaYco) [inline index.html]: - [ ] while these _are_ images, they are also links

- [ ] the text in your demo is different from the page provided 


| Yours | Original |
| --- | --- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            <figure class="dfooter-img">
+                <a href="#"><img src="images/footer2.jpg" alt="iPhone 5 colorful-image"></a> 
+            </figure>   
+            <figure class="dfooter-img">
+                <a href="#"><img src="images/footer3.jpg" alt="mac brand icon"></a> 
+            </figure> 
+            <figure class="dfooter-img">
+                <a href="#"><img src="images/footer4.jpg" alt="mac ipad image"></a> 
+            </figure> 
+        </div>
```
Comment (SpaYco) [inline index.html]: - [ ] [Optional] This isn't part of the footer but in the main

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+  border: 1px solid #858585;
+  width: 7vw;
+  padding: 11.5px;
+  height: 6vh;
+}
+
+.bg-img {
+  background: url("hero.jpg") 55% 100% no-repeat;
+  min-height: 69vh;
+}
```
Comment (SpaYco) [inline css/style.css]: - [ ] This isn't loading, use `"../images/hero.jpg"` to fix it

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] You have a warning in the [HTML Validator](https://validator.w3.org/#validate_by_input), please fix it.

image

- [ ] Elements should be aligned with [float](https://css-tricks.com/all-about-floats/) **and** [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) _or_ [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/))

- [ ] Your page doesn't seem to be fully aligned together, some sections are wider than the rest, please make them all the same width just like the original page ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by enaburekhan** [comment]
**Code review changes effected:**

- Fix html validation warning
- Align images
- Add links to main images
- Fix demo links
- Fix main section layout
- Fix background images to load correctly

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 


Good work! there are two changes you need to do and you're good to go!

## Required Changes ♻️ 

- [ ] sorry I haven't noticed this earlier, but now that the background is fixed, I see that you're not really using it and nothing changes without it, remove the  tag that's making the background, and make that CSS property as the background ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/background_image_requirement.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-backgrounds-and-gradients))  The CSS background should `cover` the whole page irrespective of the screen size. This will be a learning opportunity for the students to use the background size and position properties. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/background_size_requirement.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/))

> This is a very important step as this project focuses on building with Backgrounds and Gradients

- [ ] You have files that you're not using anymore, please delete them ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by enaburekhan** [comment]
**Code review changes effected:**

- Fix background image to look the same with the original
original-apple-image

- Remove unused images 

**Attached is the background image and layout from the original page through odin project. This is what we cloned.**

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 

The page looks great, and I can see that you have implemented all that was previously requested.

You can move on to the next projects :ok: 

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**